packages/react-components/react-list-preview/library/src/components/ListItem/useListItem.tsx

```diff
@@ -175,7 +176,7 @@ export const useListItem_unstable = (
   });
 
   const tabsterAttributes = useMergedTabsterAttributes_unstable(
-    focusableItems ? arrowNavigationAttributes : {},
```

Comment:
this was breaking build

Reply:
This was modified here: https://github.com/microsoft/fluentui/pull/32300/files#diff-d3287bdc67e7afdeb4149b129ca698a5a9eb873eebee2cb3fb998a99d7e46a79

I'm a little surprised with this 👀👀, I'll investigate

Reply:
resolved, this was my dependencies not building properly :)